schema.sql

```diff
@@ -9,4 +9,12 @@ vet_clinic=# CREATE TABLE animals(
   
   //ADD COLUMN
```

Comment:
- [ ] [Optional] This is not the correct way of writing comments in `sql`, you will need to learn how to write properly comments in sql, you can read this article: [SQL Comments](https://www.w3schools.com/sql/sql_comments.asp), for more info, after you learn that I would suggest you fix all of the comments written on this file to avoid issues with running your queries!